public/firefox.html

```diff
@@ -3,9 +3,18 @@
 
 {% block instructions %}
   <ol class="bullets-light">
-    <li><span>Download and launch <a href="https://www.mozilla.org/en-US/firefox/new/?scene=2"><strong>Firefox</strong></a>.</span></li>
-    <li data-headset="oculus_rift"><span>Ensure that your <em>Oculus Home</em> settings allow for <a href="https://blog.mozvr.com/oculus-home-rift-cv1-webvr/#enablingunknownsources">Unknown Sources</a>.</span></li>
-    <li><span>For PCs (especially laptops) using NVIDIA chipsets, there are several methods of <a href="https://alteredqualia.com/texts/optimus/">enabling the discrete GPU</a>. One way is from the <a href="https://www.nvidia.com/Download/Find.aspx">NVIDIA Control Panel</a>: Load <code>3D Settings</code> <code>&gt;</code> <code>Manage 3D Settings</code> <code>&gt;</code> <code>Program Settings</code> tab, and select <strong>Mozilla Firefox (<code>firefox.exe</code>)</strong> (with the Firefox icon) as the program to customize, and select <strong>&ldquo;High-performance NVIDIA processor&rdquo;</strong> as the preferred graphics processor.</span></li>
+    <li><span>Download and launch <a href="{{ browser.url }}"><strong>Firefox</strong></a>.</span></li>
+    <li id="setup-oculus-unknown-sources" data-headset="oculus_rift"><span>
+      <p>For the <strong data-headset="oculus_rift">Oculus Rift</strong>,
+      enable <strong>Unknown Sources</strong> in your Oculus Settings.</p>
+      <p>Launch the <strong data-headset="oculus_rift">Oculus desktop app</strong> and navigate to <code>Settings &gt; General &gt; Unknown Sources</code>.</p>
+      <p><img class="db" src="/oculus_rift/oculus-unknown-sources-settings.gif" alt="Animation of Oculus 'Unknown Sources' instructions"></p>
```

Comment:
Nice.  Much better than linking to that old page.

Reply:
See issue https://github.com/immersive-web/webvrrocks/issues/281  I included a updated gif there.

Reply:
![gif](https://user-images.githubusercontent.com/1560884/46511364-22da7780-c803-11e8-97e7-40d260de5c9c.gif)